diode-napalm-agent/diode_napalm/discovery.py

```diff
@@ -4,13 +4,39 @@
 
 import logging
 
+import importlib_metadata
 from napalm import get_network_driver
 
 # Set up logging
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-supported_drivers = ["ios", "eos", "junos", "nxos"]
+
+def napalm_driver_list():
```

Comment:
nit: return type hint
```suggestion
def napalm_driver_list() -> list[str]:
```